.eslintrc.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
+  env: {
+    'node': true
+  },
+  extends: [
+    'standard'
+  ],  
+  rules: {
+    'array-bracket-spacing': [2, 'never'],
+    'block-scoped-var': 2,
+    'brace-style': [2, '1tbs'],
```

Comment:
An other one for you to decide is "brace-style"

current:
```
try {
  somethingRisky();
} catch(e) {
  handleError();
}
```

changed to`'brace-style': [2, "stroustrup", { "allowSingleLine": true }],`
```
try {
  somethingRisky();
}
catch(e) {
  handleError();
}
```